src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -137,13 +137,13 @@ public static object GetUninitializedObjectWithFormatterServices(Type type)
         private static SerializationContext serializationContext;
 
         private static SerializationContext CurrentSerializationContext
-            => serializationContext ?? (serializationContext = new SerializationContext());
+            => serializationContext ?? (serializationContext = new SerializationContext(null));
         
         [ThreadStatic]
         private static DeserializationContext deserializationContext;
 
         private static DeserializationContext CurrentDeserializationContext
-            => deserializationContext ?? (deserializationContext = new DeserializationContext());
+            => deserializationContext ?? (deserializationContext = new DeserializationContext(null));
```

Comment:
It's not error prone, passing `null` there isn't an issue because `DeserializationContext` will coalesce the value with `RuntimeClient.Current` upon usage. It's ugly and I don't like it, but the fix is in the PR queue